world.js

```diff
@@ -781,7 +782,31 @@ const _connectRoom = async (roomName, worldURL) => {
 
 const objects = [];
 world.getObjects = () => objects;
-world.addObject = (contentId, parentId = null, position = new THREE.Vector3(), quaternion = new THREE.Quaternion(), options = {}) => {
+world.addObject = async (contentId, parentId = null, position = new THREE.Vector3(), quaternion = new THREE.Quaternion(), options = {}) => {
+  (async () => {
+    const token = await contracts.sidechain.NFT.methods.tokenByIdFull(parseInt(contentId)).call();
```

Comment:
It might make more sense to put this as part of the backend metadata fetch. It would probably be much faster.

Reply:
Actually, could already pull owner from https://tokens.webaverse.com/16 for example -- would it be a good idea to add the `monetizationPointer` as well?

Reply:
b193246, 2c57e49, and https://github.com/exokitxr/exokit-backend/pull/34